src/main/java/net/minecraftforge/network/PlayMessages.java

```diff
@@ -142,7 +142,11 @@ public static void handle(SpawnEntity msg, Supplier<NetworkEvent.Context> ctx)
                     return;
                 }
 
-                e.setPos(msg.posX, msg.posY, msg.posZ);
+                /*
+	            * Sets the postiion on the client, Mirrors what
+	            * Entity#recreateFromPacket and LivingEntity#recreateFromPacket does.
+	            */
+                e.syncPacketPositionCodec(msg.posX, msg.posY, msg.posZ);
```

Comment:
Formatting: 4 spaces without any tabs.
```suggestion
                /*
                 * Sets the postiion on the client, Mirrors what
                 * Entity#recreateFromPacket and LivingEntity#recreateFromPacket does.
                 */
                e.syncPacketPositionCodec(msg.posX, msg.posY, msg.posZ);
```